openlibrary/templates/type/author/edit.html

```diff
@@ -109,7 +111,7 @@ <h1>$_("Edit Author")</h1>
                         <div class="label">Identifiers</div>
                         <div id="hiddenIdentifierInputs"></div>
                         <div id="identifiers-display">
-                            $:render_component('AuthorIdentifiers', attrs=dict(assigned_ids_string=dict(page.remote_ids),output_selector='#hiddenIdentifierInputs',author_config_string=dict(get_author_config())))
+                            $:render_component('AuthorIdentifiers', attrs=dict(assigned_ids_string=dict(page.remote_ids),output_selector='#hiddenIdentifierInputs',author_config_string=dict(get_author_config())))<a href="https://openlibrary.org/help/faq/editing.en#author-identifiers-purpose"><i class="fa fa-question-circle" style="font-size:24px"></i></a>
```

Comment:
The Font Awesome icon can be replaced with an icon that we are already using (like [this](https://github.com/internetarchive/openlibrary/blob/master/static/images/icons/icon_help.png)).